ansible/roles_ocp_workloads/ocp4_workload_dso/tasks/infrastructure.yml

```diff
@@ -65,11 +65,21 @@
   become: true
 
 - name: Add quay route to insecure registries
-  command: "{{ ocp4_dso_openshift_cli }} patch image.config.openshift.io/cluster -p '{ \"spec\" : { \"registrySources\" : { \"insecureRegistries\" : [\"quayecosystem-quay-quay-enterprise.{{ route_subdomain }}\"] }}}' --type=merge"
+  command: >-
+    {{ ocp4_dso_openshift_cli }} patch image.config.openshift.io/cluster
+    -p '{ \"spec\" : { \"registrySources\" : { \"insecureRegistries\" :
+    [\"quayecosystem-quay-quay-enterprise.{{ route_subdomain }}\"] }}}'
+    --type=merge
```

Comment:
Check the `\"` escapes, these quotes do not need the backslash with the YAML blockquote. Compare with yaml-to-json conversion.